react/InputPassword/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import cx from 'classnames'
+import { default as LabelStyles } from '../Label/styles.styl'
+import Input from '../Input'
+import styles from './styles.styl'
+
+class InputPassword extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { visible: false }
+    this.askShowHide = props.askShowHide !== false
```

Comment:
I think `visibilityButton` (`true`/`false`) is a better name for this prop. What do you think ?

By the way it should be only taken into account in the render method.